src/main/java/uk/gov/companieshouse/service/SuppressionService.java

```diff
@@ -19,12 +20,19 @@
 
     private final SuppressionMapper suppressionMapper;
     private final SuppressionRepository suppressionRepository;
+    private final EmailService emailService;
+
     private static final Logger LOGGER = LoggerFactory.getLogger(SuppressionService.class);
 
     @Autowired
-    public SuppressionService(SuppressionMapper suppressionMapper, SuppressionRepository suppressionRepository) {
+    public SuppressionService(
+        SuppressionMapper suppressionMapper,
+        SuppressionRepository suppressionRepository,
+        EmailService emailService
+    ) {
         this.suppressionMapper = suppressionMapper;
         this.suppressionRepository = suppressionRepository;
+        this.emailService = emailService;
     }
```

Comment:
the formatting doesn't seem quite right
```suggestion
    public SuppressionService(SuppressionMapper suppressionMapper, SuppressionRepository suppressionRepository, EmailService emailService) {
        this.suppressionMapper = suppressionMapper;
        this.suppressionRepository = suppressionRepository;
        this.emailService = emailService;
    }
```